applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_ctlr.c

```diff
@@ -158,14 +296,18 @@ int bt_vol_ctlr_up(void)
 {
 	int ret;
 
-	for (int i = 0; i < ARRAY_SIZE(vcs_client_peer); i++) {
-		if (vcs_client_peer[i] != NULL) {
-			ret = bt_vcp_vol_ctlr_unmute_vol_up(vcs_client_peer[i]);
-			if (ret) {
-				LOG_WRN("Failed to volume up for remote channel %d, ret = "
-					"%d",
-					i, ret);
-			}
+	for (size_t i = 0; i < ARRAY_SIZE(vcs_client_peer); i++) {
+		if (!vcs_peer_alive(vcs_client_peer[i].peer)) {
+			continue;
+		}
+
+		int vol = vcs_client_peer[i].master_vol + CONFIG_BT_AUDIO_VOL_STEP_SIZE;
+
+		vol = CLAMP(vol, 0, UINT8_MAX);
+		LOG_DBG("bt_vol_ctlr_up for index %u vol: %u", i, vol);
+		ret = bt_vcp_vol_ctlr_set_vol(vcs_client_peer[i].peer, vol);
```

Comment:
If `vol == vcs_client_peer[i].master_vol` we don't need to set the volume, right? 

Ditto for the volume down and mute

Reply:
Isn't the mute a separate parameter? So the vol has nothing to do with mute

Reply:
w.r.t mute and unmute, if a device is already muted, then there's no point in sending the "mute" opcode to the server, and similar to unmute. 

Basically my comment referred to that if we know that the command will have no state change, then we don't need to send the command.